frontend/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>url shortener</title>
+  <meta name="viewport" content="width=device-width" />
+  <link
+      rel="stylesheet"
+      href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+
+<div class="container">
+  <div class="bar">
+
+    <div class="mdc-textfield mdc-textfield--upgraded">
+      <input type="text" id="my-textfield" class="urltochange mdc-textfield__input">
+      <label class="mdc-textfield__label mdc-textfield__label--float-above" for="my-textfield">
+        URL to shorten
+      </label>
+    </div>
+
+    <button type="submit" class="button">shorten</button>
+  </div>
+</div>
+<div class="resultUrl"></div>
+<style>
+
+html {
+  background: #e0e0e0;
+  background-size:cover;
+  font-family: 'Open Sans', sans-serif;
+  text-align: center;
+  font-size: 14px;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  margin-top: 100px;
+}
+
+.mdc-textfield__input {
+  width: 280px;
+}
+
+button {
+  background: none;
+  border: 0;
+  cursor: pointer;
+  color: white;
+  font-size: 1.1rem;
+  text-transform: uppercase;
+  background: #2196f3;
+  border-bottom: 3px solid rgba(0, 0, 0, 0.2);
+  border-right: 1px solid rgba(0, 0, 0, 0.2);
+
+}
+
+.resultUrl {
+    margin-top: 20px;
+    font-size: 2rem;
+}
+
+.smallText {
+    font-size: 0.75rem;
+    color: #757575;
+}
+
+</style>
+<script>
+const inputField = document.querySelector('.urltochange');
+const shortenButton = document.querySelector('.button');
+const requestURL = "/encode/"
+const xhr = new XMLHttpRequest();
```

Comment:
You start there by using ECMA6, so why not using directly the new [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) which was introduced in ECMA6?

Reply:
@maxibanki Good point, thanks for bringing this up! 
I'm fairly new to JS so I don't know all the tools yet. I will take a look and see how to make this change.